owasp-top10-2017-apps/a7/streaming/README.md

```diff
@@ -0,0 +1,91 @@
+ # Streaming
+
+<p align="center">
+    <img src="images/banner.png"/>
+</p>
+
+Streaming is a Angular/Spring Boot app that contains an example of multiple Cross-Site Scripting vulnerabilities and its main goal is to describe how a malicious user could exploit them on this purposefully vulnerable app.
+
+## Index
+
+- [Definition](#what-is-cross-site-scripting)
+- [Setup](#setup)
+- [Attack narrative](#attack-narrative)
+- [Objectives](#secure-this-app)
+- [Solutions](#pr-solutions)
+- [Contributing](#contributing)
+
+## What is Cross-Site Scripting?
+
+XSS flaws occur whenever an application includes untrusted data in a new web page without proper validation or escaping, or updates an existing web page with user-supplied data using a browser API that can create HTML or JavaScript. XSS allows attackers to execute scripts in the victim’s browser which can hijack user sessions, deface web sites, or redirect the user to malicious sites.
+
+The main goal of this app is to discuss how **Cross-Site Scripting** vulnerabilities can be exploited and to encourage developers to send secDevLabs Pull Requests on how they would mitigate these flaws.
+
+## Setup
+
+To start this intentionally **insecure application**, you will need [Docker][Docker Install] and [Docker Compose][Docker Compose Install]. After forking [secDevLabs](https://github.com/globocom/secDevLabs), you must type the following commands to start:
+
+```sh
+cd secDevLabs/owasp-top10-2017-apps/a7/streaming
+```
+
+```sh
+make install
+```
+
+Then simply visit [localhost:10007][App] ! 😆
+
+## Get to know the app
+
+When you access the Streaming application you will be identified as anonymous user and can to watch a streaming on registered users channel and interact with another users (or the master channel) through messages on chat.
```

Comment:
Would you mind fixing this typo:
`When you access the Streaming application you will be identified as anonymous user and can to watch a streaming on registered users channel and interact with another users (or the master channel) through messages on chat.`

To

`When accessing the Streaming application, you will be identified as an anonymous user to watch a stream on registered users channels and interact with other users (or the master channel) through messages in the chat.`